docs/documentation/server_admin/topics/identity-broker/post-login-flow.adoc

```diff
@@ -0,0 +1,44 @@
+[[_identity_broker_post_login_flow]]
+
+=== Post login flow
+
+Post login flow is useful for the situations, when you want to trigger some additional authentication actions after every login with the particular identity provider.
+For example you may want to trigger 2-factor authentication after every login of {project_name} to `Facebook` because `Facebook` does not provide 2-factor authentication during their login.
```

Comment:
```suggestion
For example, you may want to trigger 2-factor authentication after every login of {project_name} to `Facebook` because `Facebook` does not provide 2-factor authentication during its login.
```